src/Utils/TempDirectoryCreator.php

```diff
@@ -9,29 +9,40 @@
 
 namespace Infection\Utils;
 
+use Infection\Filesystem\Filesystem;
+
 class TempDirectoryCreator
 {
+    const BASE_DIR_NAME = 'infection';
```

Comment:
do we want it to be re-used? Otherwise we might as well add `/** @private */` here as we are not in PHP 7.1 hence cannot do `private const X`

Reply:
I'm using it in the tests to remove test directory. I'll add `@private` annotation anyway